src/azurelinux/manifest.json

```diff
@@ -345,7 +345,8 @@
               "os": "linux",
               "osVersion": "azurelinux3.0",
               "tags": {
-                "azurelinux-3.0-net9.0-android-openssl": {}
+                "azurelinux-3.0-net9.0-android-openssl": {},
```

Comment:
A comment might be useful as a reminder to clean this up during the next version upgrade.  Often times Dockerfiles and their associated manifest entries are copied during version upgrades.  It would be nice to get this cleanup up during that time.

Reply:
I thought about that but wasn't sure whether comments were supported in this .json

Reply:
hm looks like this file is parsed with json.net and I don't see the comment handling opt-in so I'm not sure it'd work